### ISSUE TITLE
Migration from Microsoft.Azure.ServiceBus.EventProcessorHost to Microsoft.Azure.EventHubs.Processor should work without problems.

### ISSUE THREAD

**Issue by Vossekop** [Azure/azure-event-hubs-dotnet#181]
## Steps
Migrate the code in a solution from the earlier ServiceBus Nuget package to Microsoft.Azure.EventHubs.Processor. Then start the solution.

## Actual Behavior
Processing does not start (probably because the Owner property in the partition lease info has a value of null)

## Expected Behavior
Event processing continues where it left of previously.

## Versions  
- Microsoft.Azure.ServiceBus.EventProcessorHost: v3.1.1
- Microsoft.Azure.EventHubs.Processor: v1.0.1

I have noticed that EventHubs.Processor uses an empty string as the value in the partition lease info blobs instead of null. Might this be the problem here?

**Comment by serkantkaraca** [comment]
I will try the scenario and update the thread.

**Comment by serkantkaraca** [comment]
I was not able to reproduce the reported issue. Below is my scenario that successfully completed. Can you provide more details what kind of issue you were hitting? Any errors you observed during migration? Please provide a sample project if you can reproduce.

1. Create an EevntHub with 2 partitions.
2. Create a project to send and receive messages on ServiceBus .NET client. Receive with EPH.
3. Run the project and send 10 messages.
4. Register and run processor host and receive 10 messages back and checkpoint.
5. Run client multiple times to confirm that checkpoint holds.

6. Migrate project to .NetCore client. Make proper changes to build. Don't change lease container name.
7. Run the project and let it send 10 more messages.
8. Confirm that EPH is able to receive those 10 new messages only.

**Comment by Vossekop** [comment]
Since I first posted this issue I've had contact with Microsoft Support about an issue we were having with our event hub. As far as we could determine there is a bug in EventProcessorHost v3.1.1 that caused some weird behavior for us (processor host restarting in a loop with lease lost exceptions continuously).

Because of these issues (but before I knew the cause) I tried to upgrade to the .NetCore client, to see if that helped and thats when I seemed to have a migration issue. I now believe the blob lease issues from the earlier version caused some problems for me. I have since downgraded to an earlier version of EventProcessorHost that is not causing problems for me. And I will upgrade my solution to work with functions on the eventhub skipping the need to do any migration to the .NetCore client again.

I did not react sooner since I believed there was still a migration issue because I noticed a full typename in the lease blob file when I opened it (generated by Newtonsoft no doubt). I know Newtonsoft has global settings that control this behavior, I can imagine deserializing the blob when it contains the old type name might cause some issues.
